<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SFTPBrowser</class>
 <widget class="QWidget" name="SFTPBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SFTP Browser</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="toolbar" native="true">
     <layout class="QHBoxLayout" name="toolbarLayout">
      <item>
       <widget class="QPushButton" name="connectButton">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="separator1">
        <property name="maximumSize">
         <size>
          <width>2</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="uploadButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>‚¨ÜÔ∏è Upload</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="downloadButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>‚¨áÔ∏è Download</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="separator2">
        <property name="maximumSize">
         <size>
          <width>2</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>üóëÔ∏è Delete</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="newFolderButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>üìÅ New Folder</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="separator3">
        <property name="maximumSize">
         <size>
          <width>2</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="refreshButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>üîÑ Refresh</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="toolbarSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="mainSplitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="browserSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="localPanel" native="true">
       <layout class="QVBoxLayout" name="localLayout">
        <item>
         <widget class="QLabel" name="localPathLabel">
          <property name="text">
           <string>Local Files</string>
          </property>
          <property name="styleSheet">
           <string notr="true">font-weight: bold; padding: 5px;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeView" name="localTreeView">
          <property name="dragEnabled">
           <bool>true</bool>
          </property>
          <property name="dragDropMode">
           <enum>QAbstractItemView::DragOnly</enum>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="remotePanel" native="true">
       <layout class="QVBoxLayout" name="remoteLayout">
        <item>
         <widget class="QLabel" name="remotePathLabel">
          <property name="text">
           <string>Remote Files - Not Connected</string>
          </property>
          <property name="styleSheet">
           <string notr="true">font-weight: bold; padding: 5px;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="remoteListWidget">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="acceptDrops">
           <bool>true</bool>
          </property>
          <property name="dragDropMode">
           <enum>QAbstractItemView::DropOnly</enum>
          </property>
          <property name="showDropIndicator" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="transferQueueContainer" native="true">
      <layout class="QVBoxLayout" name="transferQueueLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
